interp/src/debugger/debugger_core.rs

```diff
@@ -175,6 +198,28 @@ impl<C: AsRef<Context> + Clone> Debugger<C> {
             .collect_vec();
         self.manipulate_breakpoint(Command::Delete(parsed_bp_ids));
     }
+
+    pub fn cont(&mut self) -> StoppedReason {
```

Comment:
as you point out this should really return a result type. If you don't want to
handle that at least unwrap the error from `self.do_continue` rather than
dropping it on the floor

Reply:
updated this, what should the adapter return in the case of an error? Not sure if it should terminate or just log an error (currently stubbed in code for logging an error)